pages/deployments.md.erb

```diff
@@ -0,0 +1,146 @@
+
+Creating deployment pipelines makes it easier to:
+
+* Separate deployment failures from test failures
+* Separate test and deployment pipeline.yml files
+* Re-run failed deployments
+* Simplify adding rollback steps
+* Group other deploy-related tasks with the deployment steps
+* Use teams for role based access control
+* Whitelist deploy pipelines in agent hooks
+
+For example, the following "My App" test pipeline will trigger a build on the "my-app-deployment" pipeline whenever a test build passes on `master`:
+
+```yml
+steps:
+  - label: ":hammer:"
+    command: "scripts/test"
+
+  - wait
+
+  - label: ":rocket:"
+    trigger: "my-app-deployment"
+    branches: "master"
+```
+
+The deployment pipeline is configured to pipeline upload the following  file:
+
+The "my-app-deployment" pipeline below is defined in the `.buildkite/pipeline.deploy.yml` file, and will run the `deploy` script when a build is triggered by the "my-app-test" pipeline above. 
+
+```yml
+steps:
+  - label: ":rocket:"
+    command: "scripts/deploy"
+    branches: "master"
+    concurrency_group: "my-app-deploy"
+    concurrency_limit: 1
+```
+
+The ":rocket:" step in the deployment example above uses the concurrency attributes `concurrency_group` and `concurrency_limit`. These work together to ensure that no matter how many builds are triggered on this pipeline at once, no ":rocket:" steps can run in parallel. It also prevents deploys clashing and attempting to write to the same resources. 
+
+<screenshot of a deploy pipeline>
+
+## Manual approval steps
+
+Adding a manual approval to your pipeline before your deployment ensures that a deploy never goes out without explicit approval.
+
+To create a manual approval for your deploy, add a block step before your deploy step or before the step that triggers your deploy pipeline. 
+
+The below example uses the same pipeline as the [Single deployment step](/docs/deployments#single-deployment-steps) section, but adds a block step before the step that performs the deploy:
+
+```
+steps:
+  - label: ":hammer:"
+    command: "scripts/test"
+  
+  - block: "Unblock deployment?"
+  
+  - label: ":rocket:"
+    command: "scripts/deploy"
+    branches: "master"
+    concurrency_group: "my-app-deploy"
+    concurrency_limit: 1
+```
+
+Until the block step is manually unblocked either in Buildkite or via an API call, the build will be paused and the ":rocket:" deployment step will not run.
+
+## External deployment systems
+
+As well as writing deployment scripts, you can use any external systems alongside Buildkite to perform your deployments.
+
+### Container services
+
+You can deploy images or containers to services like Kubernetes, Heroku, ECS, or Github Deployments. Deploy your containers directly from a script in a Buildkite command step using either the single deploy step pattern or the dedicated pipeline pattern discussed above.
```

Comment:
GitHub Deployments isn't related to container services, and Heroku deployments don't usually use containers either. Perhaps something like:

```suggestion
You can deploy images or containers to services like Kubernetes, Heroku, or ECS from a script in a Buildkite command step, similar to how you'd do it on a command line.
```